packages/e2e-tests/config/setup-performance-test.js

```diff
@@ -17,7 +17,7 @@ import {
 const PUPPETEER_TIMEOUT = process.env.PUPPETEER_TIMEOUT;
 
 // The Jest timeout is increased because these tests are a bit slow
-jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
+jest.setTimeout( PUPPETEER_TIMEOUT || 10000000 );
```

Comment:
Isn't this a really big timeout? Do we need it to be that big ( almost 3 hours )? I do not know the  `PUPPETEER_TIMEOUT ` env variable's value though..

Reply:
I think we don't really care about the timeout for the performance tests. They can take long if needed, what's important is the result of the each typing key... The tests take a long time because they insert tousands of blocks and repeat the typing operation multiple times (same for selection)

Reply:
Now that the actual performance issue is fixed on master, the previous timeout should be sufficient though, so I restored it.